randovania/games/samus_returns/logic_database/Area 6.txt

```diff
@@ -20,7 +20,7 @@ Extra - asset_id: collision_camera_034
   > Before Diggernaut Area
```

Comment:
this node should get renamed, i dont like this name. `in front of diggernaut arena` or `left hill` or something